spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -194,14 +194,15 @@
     end
 
     context 'LOA1 user' do
-      it 'hashes password exactly once' do
+      it 'hashes password exactly once, hashes email access key exactly once' do
         allow(FeatureManagement).to receive(:use_kms?).and_return(false)
         encrypted_key_maker = EncryptedKeyMaker.new
         allow(EncryptedKeyMaker).to receive(:new).and_return(encrypted_key_maker)
         user = create(:user, :signed_up)
 
-        expect(UserAccessKey).to receive(:new).exactly(:once).and_call_original
-        expect(encrypted_key_maker).to receive(:unlock).exactly(:once).and_call_original
+        expect(UserAccessKey).to receive(:new).exactly(:twice).and_call_original
+        expect(encrypted_key_maker).to receive(:unlock).exactly(:twice).and_call_original
+        expect(EncryptedEmail).to receive(:user_access_key).exactly(:once).and_call_original
```

Comment:
I think with the updated method name, this spec would reveal more information: that calling the class method is what hashes the access key.